src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -542,6 +498,8 @@ void AlgorithmPaintToolBox::updateMagicWandComputation()
 
 void AlgorithmPaintToolBox::activateStroke()
 {
+    m_wandInfo->hide();
```

Comment:
I've rebased this PR, and added this line which allow to hide the wand label ("Select a pixel in the image blablabla") when we click on Paint/Erase.